<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fifo</class>
 <widget class="QWidget" name="fifo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F I F O(先进先出页面置换算法)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 0px;
background-color:white;

</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>请 输 入 块 数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>请 输 入 页 面 数 量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>请 输 入 访 问 序 号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font-family:宋体;
font-size:30px;
background-color:rgb(160,160,160);
</string>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
font-family:宋体;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>结 果：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 12px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>861</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color:rgb(73,143,242,38%);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>650</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 30px;
font-family:宋体;
font-size:30px;
background-color:rgb(160,160,160);
</string>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>fifo</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
